app/controllers/amend/ConfirmSubmitAFTAmendmentController.scala

```diff
@@ -106,7 +106,8 @@ class ConfirmSubmitAFTAmendmentController @Inject()(override val messagesApi: Me
       val previousVersion = amendedVersion - 1
 
       aftConnector.getAftOverview(pstr, Some(localDateToString(startDate)), Some(Quarters.getQuarter(startDate).endDate)).flatMap { seqOverview =>
-        val isCompilable = seqOverview.exists(_.compiledVersionAvailable == true)
+        val isCompilable = seqOverview.filter(_.versionDetails.isDefined)
```

Comment:
Method longer than 50 lines. Also, don't need the == true bit (just remove it)

Reply:
Split method and removed ==true